src/scroll-timeline-css.js

```diff
@@ -31,8 +31,10 @@ function initMutationObserver() {
    * @param {HtmlStyleElement} el style tag to be parsed
    */
   function handleStyleTag(el) {
-    // Don’t touch empty style tags.
-    if (el.innerHTML.trim().length === 0) {
+    // Don’t touch empty style tags nor tags controlled by aphrodite.
+    // Details at https://github.com/Khan/aphrodite/blob/master/src/inject.js,
+    // but any modification to the style tag will break the entire page.
+    if (el.innerHTML.trim().length === 0 || 'aphrodite' in item.dataset) {
```

Comment:
I think s/item/el ? I don't see item defined

Reply:
omg Tuesday is getting me, and my over-reliance on typescript. Thank you, updated!